.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+_Before you submit a pull request, please make sure you have to following:_
+
+- [ ] The title of this pull request offers a good description of what is changed (as it is used in release notes).
+- [ ] Your branch follows the [GitFlow](https://datasift.github.io/gitflow/IntroducingGitFlow.html) branching model.
+- [ ] The code follows the [best practices](#).
+- [ ] The functionality has been manually tested (if applicable)
+- [ ] A new unit test has been created, or the existing test has been updated.
+- [ ] All new and existing tests passed.
+- [ ] The module version has been bumped.
```

Comment:
I would highlight that the version should be bumped in the `manifest.php` and also the `Updater.php`, following the semantic versioning.

Reply:
I would also add a new point here to list all the other pull requests which are depending on this one or connected to this modification.